README.md

```diff
@@ -49,6 +49,7 @@ When looking for support, please first search for your question in these venues:
 * [Node.js Help][]
 * [Open or closed issues in the Node.js GitHub organization](https://github.com/issues?utf8=%E2%9C%93&q=sort%3Aupdated-desc+org%3Anodejs+is%3Aissue)
 * [Questions tagged 'node.js' on StackOverflow][]
+* [Node.js Slack Community](node-js.slack.com): Visit [nodeslackers.io](nodeslackers.io) to register 
```

Comment:
1. `node-js.slack.com` -> `https://node-js.slack.com/`
Without protocol part, URL is relative and 404.

2. `nodeslackers.io` -> `https://nodeslackers.io/` ?
However, both https://nodeslackers.io/ and http://nodeslackers.io/ is `nodeslackers.io’s server IP address could not be found` for me.
Should it be http://www.nodeslackers.com/ ?

Reply:
Looks like it! I will update. Thanks.

Reply:
should probably just serve on root of domain and alias `www` to `nodeslackers.io`

Reply:
Should be! Good catch. Trying to improve this short term.